config-object-store/src/main/java/org/hypertrace/config/objectstore/IdentifiedObjectStore.java

```diff
@@ -114,6 +116,33 @@ public List<ContextualConfigObject<T>> getAllObjects(RequestContext context) {
                 Collectors.toUnmodifiableList(), this::orderFetchedObjects));
   }
 
+  public List<ContextualConfigObject<T>> getAllObjectsWithFilter(
```

Comment:
This is specific to the push down impl and should be a private method there. We don't want to expose the internals to the consumers of the object store

Reply:
Updated to package private

Reply:
But why put it in the parent class to begin with? Is some other code calling it? If not, we should make it fully private it and move it into the filter impl object store.